src/App.js

```diff
@@ -1,26 +1,68 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
 
-function App() {
+import Board from './components/Board';
+
+const PLAYER_1 = 'x';
+const PLAYER_2 = 'o';
+
+const generateSquares = () => {
+  const squares = [];
+  let row = 0;
+  let col = 0;
+  let currentId = 0;
+  console.log(`row = ${ row }`);
+
+  while (row < 3) {
+    squares.push([]);
+    while (col < 3) {
+      squares[row].push({
+        id: currentId,
+        value: '',
+      });
+      col += 1;
+      currentId += 1;
+    }
+    row += 1;
+    col = 0;
+  }
+
+  return squares;
+}
```

Comment:
It's more idiomatic to use `for` loops here.

```suggestion
const generateSquares = () => {
  const squares = [];
  let currentId = 0;
  console.log(`row = ${ row }`);

  for (let row = 0; row < 3; row++) {
    squares.push([]);
    while (let col = 0; col < 3; col++) {
      squares[row].push({
        id: currentId,
        value: '',
      });
      currentId += 1;
    }
  }

  return squares;
}
```